ironmon_tracker/Program.lua

```diff
@@ -448,8 +448,8 @@ function Program.autoTrackAbilitiesCheck(battleMsg, enemyAbility, playerAbility)
 	local battler = Memory.readbyte(GameSettings.gBattleScriptingBattler) -- 0 or 2 if player, 1 or 3 if enemy
 	local attacker = Memory.readbyte(GameSettings.gBattlerAttacker) -- 0 or 2 if player, 1 or 3 if enemy
 	local battlerTarget = Memory.readbyte(GameSettings.gBattlerTarget)
-	local moveFlags = Memory.readbyte (GameSettings.gMoveResultFlags)
 	local currentTurn = Memory.readbyte(GameSettings.gBattleResults + 0x13)
+	local levitateCheck = Memory.readbyte(GameSettings.gBattleCommunication + 0x6)
```

Comment:
Can you wrap this inside of the if-block that checks if the enemyAbility is Levitate? This way, we don't have to do an extra memory read every call to this function, and only do it while it's possible that the enemy has this ability.

Something like:
`if ability is 26 and attacker is us then... read from memory, if check, then...`

Reply:
Sure, done